.github/workflows/release-20_build-rc.yml

```diff
@@ -69,12 +69,13 @@ jobs:
   build-polkadot-parachain-binary:
     needs: [validate-inputs]
     if: ${{ inputs.binary == 'polkadot-parachain' || inputs.binary == 'all' }}
-    uses: "./.github/workflows/release-reusable-rc-buid.yml"
+    uses: "./.github/workflows/release-reusable-rc-build.yml"
     with:
       binary: '["polkadot-parachain"]'
       package: "polkadot-parachain-bin"
       release_tag: ${{ needs.validate-inputs.outputs.release_tag }}
       target: x86_64-unknown-linux-gnu
+      features: runtime-benchmarks
```

Comment:
Polkadot parachain should not have this feature activated, as otherwise the included wasm files will have this enabled as well.

Reply:
Good catch! 🙈 Thought about enabling `polkadot-omni-node-lib/runtime-benchmarks` by default for `polkadot-parachain`, but I get multiple errors like below (for many runtimes):

```
error[E0046]: not all trait items implemented, missing: `try_successful_origin`
    --> substrate/frame/collective/src/lib.rs:1540:1
```

Not sure if we're easily able to enable the `benchmark` subcommands in `polkadot-parachain`. Need to investigate a bit more.

Reply:
In the meantime, this PR only builds `polkadot-omni-node` with `runtime-benchmarks`.